docs/scully-cmd-line.md

```diff
@@ -33,13 +34,13 @@ npx scully serve
 
 Starts the scully server. This process does not _build_ the project. It only serves the Agular build files, and the Scully static files.
 
-## noWatch
+## Watch
 
 ```bash
-npx scully --nw
+npx scully --watch
 ```
 
-By default, Scully has the watchMode active. Use this flag to run scully only once.
+By default, Scully has the watchMode in false. You need add this flag for use Watch Mode.
```

Comment:
I would suggest to change 
from : `You need add this flag for use Watch Mode.`
to : `You need to add this flag to use Watch Mode.`